db/migrate/20161208064834_create_group_histories.rb

```diff
@@ -0,0 +1,15 @@
+class CreateGroupHistories < ActiveRecord::Migration
+  def change
+    create_table :group_histories do |t|
+      t.integer :group_id, index: true, null: false
+      t.integer :acting_user_id, index: true, null: false
```

Comment:
ouch, `index: true` though sparse worries me, ideally I think we should create indexes in an extra step and name them while creating them.